src/main/java/org/sqlite/jdbc3/JDBC3ResultSet.java

```diff
@@ -589,7 +512,7 @@ public Timestamp getTimestamp(String col) throws SQLException {
      * @see java.sql.ResultSet#getTimestamp(java.lang.String, java.util.Calendar)
      */
     public Timestamp getTimestamp(String c, Calendar ca) throws SQLException {
-        return getTimestamp(findColumn(c), ca);
```

Comment:
(Duplicate) It seems like this moves the driver away from the JDBC spec by ignoring the user supplied Calendar/Timezone information.